reference/git_github.rst

```diff
@@ -3,6 +3,16 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
+Best practises
+--------------
+- Create a separate pull request for each bug. If you have multiple commits in one pull request, squash them into one.
```

Comment:
The Reference section (of which this is part) is currently aimed at all contribs, not just beginners. From the front page:

> If you’re already a Webdev contributor, you can find generally-useful info, such as style guides, in the Reference.